cypress/support/step-definitions/toast-steps.js

```diff
@@ -1,22 +1,25 @@
-import { toastPreview, toastComponent } from '../../locators/toast';
+import { toastPreview, toastComponent, toastTogglePreview } from '../../locators/toast';
 import { getDataElementByValue } from '../../locators';
 
-Then('Toast icon is set to {string}', (icon) => {
-  toastPreview().children().should('have.attr', 'type', `${icon}`)
-    .and('have.attr', 'data-element', `${icon}`)
-    .and('be.visible');
+When('I click on {string} Toggle Preview', (e) => {
+  cy.iFrame(`#${e}`).scrollIntoView();
```

Comment:
1. Could you use already defined step? 
2. Please move this locator `#${e}` in to the proper file